### DISCUSSION TITLE
Rate limit exceeded

### DISCUSSION THREAD

**Discussion by pullmann4rent** [supabase/supabase#22016] [Questions]
Hello,

I have many tests and I use supertest so make real world tests and I login often in different accounts and after 100 tests I got this by a login:

```
Error: AuthApiError: Rate limit exceeded
    at SignIn (file:///C:/Users/t/Desktop/xxx/xxx/src/controllers/auth/sign_in.ts:21:13)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
```

So how can I prevent it or can make it to a higher number ?

PS: is it IP bound or for global users ?
Because the error comes only when I too often login. But with different accounts

**Comment by GaryAustin1** [comment]
These are the auth rate limits: https://supabase.com/docs/guides/platform/going-into-prod#auth-rate-limits
Some are IP limited and some are global.

A couple can be changed here: 
![image](https://github.com/supabase/supabase/assets/54564956/98770e6a-3b90-46bd-8561-32480d71623e)

**Reply by pullmann4rent** [reply]
And which is Ip bound ? That what I described ?

**Reply by GaryAustin1** [reply]
From the doc link:
![image](https://github.com/supabase/supabase/assets/54564956/f5d93d48-c7a2-4570-8587-a545c75cfd25)
I'm going to guess you would be hitting the verify limit, but you don't show what call/calls are being made.
You can also look at the dashboard logs for auth and api edge and possibly get more info on what calls are hitting the limit.
